### PR TITLE
Sergey Budkov 2536

### PR DESCRIPTION
**Pull request by sergeybp**

### CODE REVIEW

File: laPics/app/src/main/java/lapics/sergeybudkov/ru/lapics/AllPictureShow.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.all_pictures_menu, menu);
+        return true;
+    }
+
+
+    private void loadImages() {
+        dialog = new ProgressDialog(AllPictureShow.this);
+        dialog.setCancelable(false);
+        dialog.setTitle("Загружаем картники");
```
Comment (svasilinets) [inline laPics/app/src/main/java/lapics/sergeybudkov/ru/lapics/AllPictureShow.java:100]: strings -> resources

File: laPics/app/src/main/java/lapics/sergeybudkov/ru/lapics/AllPictureShow.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+import android.widget.GridView;
+import android.widget.Toast;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AllPictureShow extends Activity {
+
+    public static int W,H;
+    private int width;
+    public static ProgressDialog dialog;
```
Comment (svasilinets) [inline laPics/app/src/main/java/lapics/sergeybudkov/ru/lapics/AllPictureShow.java:23]: it is very bad to make any UI element static, so its life is longer than life of bounded Activity

**Comment by svasilinets** [comment]
E/WindowManager﹕ android.view.WindowLeaked: Activity lapics.sergeybudkov.ru.lapics.AllPictureShow has leaked window com.android.internal.policy.impl.PhoneWindow$DecorView{1e27a9cd V.E..... R......D 0,0-1026,444} that was originally added here
            at android.view.ViewRootImpl.<init>(ViewRootImpl.java:363)
            at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:261)
            at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:69)
            at android.app.Dialog.show(Dialog.java:298)
            at lapics.sergeybudkov.ru.lapics.AllPictureShow.loadImages(AllPictureShow.java:104)
            at lapics.sergeybudkov.ru.lapics.AllPictureShow.onCreate(AllPictureShow.java:55)
            at android.app.Activity.performCreate(Activity.java:5933)
            at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1105)
            at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2251)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2360)
            at android.app.ActivityThread.access$800(ActivityThread.java:144)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1278)
            at android.os.Handler.dispatchMessage(Handler.java:102)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5221)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:899)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:694)

I wasn't able to start it neither on my nexus 5, neither on emulator, neither on genymotion.

0.

**Comment by sergeybp** [comment]
screen1
screen2

Это немного непонятно: вот как это работает на Genymotion у меня и на моем Nexus 7. Может быть я могу подойти к вам перел экзаменом(21), чтобы разобраться в чем проблема?

**Pull request closed without merging**